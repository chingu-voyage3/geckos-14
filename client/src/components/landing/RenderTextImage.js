import React from 'react';

const RenderTextImage = (props) => {
  return (
    <div className="features-two">
      <h2> {props.heading ? props.heading : true} </h2>
      <div className="features-two-container">
        <div>
          <h4> {props.text} </h4>
        </div>
        <div>
          <img src={props.image} alt={props.alt}/>
        </div>
      </div>
    </div>
  );
}

export default RenderTextImage;
