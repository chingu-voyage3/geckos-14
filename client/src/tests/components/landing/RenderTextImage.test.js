import React from 'react';
import RenderTextImage from '../../../components/landing/RenderTextImage';
import devices from '../../../assets/images/devices.png';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<RenderTextImage
    text="Easily add or remove devices in your DashThings board"
    image = {devices}
    alt = "Internet of things devices"
  />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
