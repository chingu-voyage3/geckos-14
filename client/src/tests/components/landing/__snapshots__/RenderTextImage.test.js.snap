// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<section
  className="section"
>
  <h2
    className="text-center font-weight-bold mt-10"
  >
     
     
  </h2>
  <div
    className="row"
  >
    <div
      className="col-md-6"
    >
      <h4
        className="font-weight-bold"
      >
         
        Easily add or remove devices in your DashThings board
         
      </h4>
    </div>
    <div
      className="col-md-6"
    >
      <img
        alt="Internet of things devices"
        className="img-fluid rounded"
        height="470px"
        src="devices.png"
        width="470px"
      />
    </div>
  </div>
</section>
`;
